tests/unit/test_charm.py

```diff
@@ -1570,20 +1645,22 @@ def test_ingresses_for_multiple_relations_blocked(
         rel_data["path-routes"] = "/funicorn"
         self.harness.update_relation_data(rel_id, "funicorn", rel_data)
 
-        expected_status = ActiveStatus("Ingress with service IP(s): 10.0.1.12")
+        expected_status = ActiveStatus("Service IP(s): 10.0.1.12")
```

Comment:
As above, consider just checking for key values or specific words rather than the entire service message